src/pilot/templates/nic-configs/dpdk_mode1_4_port/compute-ovs-dpdk-mode1.yaml

```diff
@@ -141,52 +141,37 @@ resources:
         os_net_config:
           network_config:
             -
-              type: ovs_bridge
-              name: br-tenant
-              use_dhcp: false
-              dns_servers: {get_param: DnsServers}
-              addresses:
+              type: linux_bond
+              name: bond0
+              bonding_options: {get_param: ComputeBondInterfaceOptions}
+              members:
                 -
-                  ip_netmask:
-                    list_join:
-                      - '/'
-                      - - {get_param: ControlPlaneIp}
```

Comment:
In the resulting file, nothing is assigned the control plane IP.  I would think this would result in no interface being on the provisioning network, and we have to have an interface on the provisioning network.